packages/fxa-content-server/app/styles/modules/_marketing.scss

```diff
@@ -20,6 +23,10 @@
     height: 45px;
     width: 152px;
   }
+
+  .clearfix::after {
```

Comment:
In my local testing this doesn't appear to do anything, I don't believe it is needed.

Reply:
@vbudhram  I made this change since clearfix::after was adding unnecessary content that was taking space and hence the badges weren't getting centered properly. By setting content to none for just this case,  it has been removed.

Reply:
Could you post a screenshot of the before/after, my eyes could be playing tricks on me?